pycytominer/tests/test_annotate.py

```diff
@@ -1,19 +1,20 @@
 import os
-import tempfile
 import random
+import tempfile
+
 import pandas as pd
 from pycytominer.annotate import annotate
 
 random.seed(123)
 
 # Get temporary directory
-tmpdir = tempfile.gettempdir()
+TMPDIR = tempfile.gettempdir()
```

Comment:
Thanks for making these changes! The test suite really does need a makeover...

Reply:
Cheers, thank you for the chance to make the changes!